src/Polysemy/Trace.hs

```diff
@@ -29,14 +33,45 @@ data Trace m a where
 makeSem ''Trace
 
 
+------------------------------------------------------------------------------
+-- | Run a 'Trace' effect by printing the messages to the provided 'Handle'.
+--
+-- @since TODO
+traceToHandle :: Member (Embed IO) r => Handle -> Sem (Trace ': r) a -> Sem r a
+traceToHandle handle = interpret $ \case
+  Trace m -> embed $ hPutStrLn handle m
+{-# INLINE traceToHandle #-}
+
+
+------------------------------------------------------------------------------
+-- | Run a 'Trace' effect by printing the messages to stdout.
+--
+-- @since TODO
+traceToStdout :: Member (Embed IO) r => Sem (Trace ': r) a -> Sem r a
+traceToStdout =
+  traceToHandle stdout
```

Comment:
Any reason these aren't one-liners?

Reply:
you mean as in unfolding `traceToHandle`?

Reply:
No, I mean
```
traceToStdout =traceToHandle stdout
```

Reply:
oh! no, I just always put a newline after the `=`. gonna fix